docs/content/DataSourceMetadataQuery.md

```diff
@@ -2,7 +2,11 @@
 layout: doc_page
 ---
 # Data Source Metadata Queries
-Data Source Metadata queries return metadata information for a dataSource. It returns the timestamp of latest ingested event for the datasource. The grammar is:
+Data Source Metadata queries return metadata information for a dataSource.  These queries return information about:
+
+* The timestamp of latest ingested event for the dataSource. This is the ingested event without any consideration of rollup.
```

Comment:
The thinking was that this query would expand eventually to include more information.